.travis.yml

```diff
@@ -31,11 +31,11 @@ script:
         gemini test test/visual;
       fi &&
       if [[ "$TEST_SUITE" = "unit_tests" ]]; then
-        xvfb-run -s '-screen 0 1024x768x24' polymer test --env saucelabs;
+        wct --env saucelabs;
```

Comment:
could you please apply the same change to date-picker to verify that viewport size doesn't matter for tests?